### DISCUSSION TITLE
How quickly does charging start when below limit?

### DISCUSSION THREAD

**Discussion by marcospgp** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1248] [Q&A]
If I set an 80% limit on battery charge, will there be a cycle where once charge gets to 79% it will charge back to 80% and so on ad infinitum?

Would like to avoid that scenario as it seems unlikely to benefit the battery's health.

Thanks!

**Comment by MatthiasKerbl** [comment]
Hi @marcospgp,

Yes, if your battery drops to 79%, it will start charging again to 80%. If you want to avoid this, you can check out AlDente Pro which has a feature called Sailing Mode where you can set a range within which the battery does not start changing again. You can read more about this feature on our [blog ](https://apphousekitchen.com/feature-explanation-sailing-mode/).

Best,
Matthias

**Comment by marcospgp** [comment]
Does that imply the free tier of AlDente can actually be detrimental to battery life?

Repetitive short lived charging cycles does not sound ideal 🤔

Seems like at minimum there should be a non-configurable interval, say of 5%, before charging resumes. So for a charge limit of 80%, charging would only resume at 75%.

**Reply by MatthiasKerbl** [reply]
Hi @marcospgp,

From our experience, it is better to hold a lower charge limit with some small charges/discharges than to not have a charge limit at all.

Best,
Matthias
